community/kernel-api/src/main/java/org/neo4j/internal/kernel/api/helpers/RelationshipDenseSelectionIterator.java

```diff
@@ -43,7 +43,7 @@ public boolean hasNext()
     {
         if ( next == RelationshipSelections.UNINITIALIZED )
         {
-            fetchNext();
+            this.next();
```

Comment:
This looks like an infinite loop to me?

Reply:
oops, intellij refactoring gone wrong...